lib/matplotlib/axis.py

```diff
@@ -657,6 +657,7 @@ def __init__(self, axes, *, pickradius=15, clear=True):
             self.clear()
         else:
             self.converter = None
+            self._explicit_converter = False
```

Comment:
Slight perference for
```suggestion
            self._converter_is_explicit = False
```
This makes it clear that the content is a bool. `_explicit_converter` could be understood to contain an explicit converter instance.